### ISSUE TITLE
Keyboard trap in date picker

### ISSUE THREAD

**Issue by renujain31** [ushahidi/platform#4190]
**Describe the bug**
When we open the date picker we can't access anything else with the keyboard

**Where was the bug observed**
in v5 deployment /views/maps

**To Reproduce**
1. Go to the maps view.
2. Click on the search input box.
3. Navigate to the date range and edit that using keyboard
4. The date picker is not accessible via keyboard

**Screenshots**
![image](https://user-images.githubusercontent.com/70752431/102891228-d632f380-4483-11eb-9baf-35095dd9293e.png)

**Comment by Angamanga** [comment]
Waiting for QA
